extensions/applicationinsights-clickanalytics-js/src/events/PageAction.ts

```diff
@@ -115,4 +133,15 @@ export class PageAction extends WebEvent {
         }
     }
 
+    private _sanitizePageActionEventContent(pageActionContent: any) {
+        if(pageActionContent) {
+            delete pageActionContent.id;
+            delete pageActionContent.parentid;
+            delete pageActionContent.parentname;
+            if(isValueAssigned(this._config.dataTags.parentDataTag)) {
```

Comment:
Can there be a case where _config or _config.dataTags is null or uindefined? If so we should guard against this throwing a JS error

Reply:
Actually by default we create an empty object this._config.dataTags